src/libsystem_networkextension/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+project(system_networkextension)
+
+cmake_minimum_required(VERSION 2.4.0)
+
+add_definitions(-DTARGET_OS_MAC=1)
+add_definitions(-D__APPLE__ -D__DYNAMIC__)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -nostdinc -D__DARWIN_UNIX03 -fPIC -w")
```

Comment:
Aren't these flags and definitions already set for all projects in main CMakeLists?

Reply:
Most of the stuff is copied from `libsystem_coreservices`'s CMakeLists.txt. I'll remove it.

Reply:
Removed it.

Reply:
> Most of the stuff is copied from libsystem_coreservices's CMakeLists.txt.

Yeah, we should clean up all the other subprojects of specifying common flags and options and includes too. Want to try doing that? :slightly_smiling_face: